<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="ATM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="welkomScherm">
    <widget class="QPushButton" name="welkom_taal_button">
     <property name="geometry">
      <rect>
       <x>264</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[language settings]</string>
     </property>
    </widget>
    <widget class="QLabel" name="welkom_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>welkom 
 voer uw pas in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="welkom_inlog_button">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>220</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pas invoer simulatie</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inlogScherm">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Voer uw pincode in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inlog_taal_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>[language settings]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inlog_hoofd_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Doorgaan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inlog_welkom_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inlog_invoerveldPincode">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="taalScherm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="taal_optieNederlands_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Nederlands</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taal_optieEngels_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>English</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taal_optieDuits_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Deutsch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taal_welkom_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>afrbreken</string>
     </property>
    </widget>
    <widget class="QPushButton" name="taal_hoofd_of_welkom_of_inlog_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>terug</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hoofdScherm">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>HoofdScherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoofd_snelPinnen_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Snel â‚¬70,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoofd_saldo_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Saldo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoofd_pinKeuze_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bedrag kiezen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoofd_taal_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Taal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hoofd_welkom_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="saldoScherm">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SaldoScherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saldo_pinKeuze_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Geld opnemen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saldo_welkom_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saldo_hoofdscherm_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoofdscherm</string>
     </property>
    </widget>
    <widget class="QLabel" name="saldo_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pinKeuzeScherm">
    <widget class="QLabel" name="Labelpinnen">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KeuzeScherm pinnen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_optie1_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚¬20,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_optie2_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚¬50,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_optie3_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚¬70,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_optie4_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚¬100,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_optie5_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚¬120,-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_zelfBedragKiezen_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zelf kiezen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_welkom_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>afbreken</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pinKeuze_hoofd_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoofdscherm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="biljetScherm">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>biljetKeuzeScherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biljetKeuze_welkom_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biljetKeuze_hoofd_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoofdscherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biljetKeuze_biljetNietBelangrijk_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Boeit niet</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biljetKeuze_biljetOpties1_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Opties biljetten</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bedragScherm">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>BedragKeuzeScherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bedragKeuze_hoofd_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bedragKeuze_welkom_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoofdscherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bedragKeuze_bevestigen_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bevestigen</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bonScherm">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>140</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bon Scherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bon_metBon_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ik wil een bon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bon_zonderBon_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ik wil geen bon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bon_hoofd_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hoofdscherm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bon_welkom_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afbreken</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verwerkingsScherm">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>verwerkingsScherm</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
